docs/quick-start.md

```diff
@@ -17,105 +18,171 @@ information at [planner usage](planner-usage.md).
+For Vagrant to work you also need a "provider" such as VirtualBox, VMware, Hyper-V.
+
+Note:
+
+- Not all provider versions and Vagrant versions are compatible!
+- We recommend using Vagrant 2.4.1 and [VirtualBox 7.0.20](https://www.virtualbox.org/wiki/Download_Old_Builds_7_0).
+
+To create your virtual machine using Vagrant create a new directory `<my-fast-downward-vm>` containing only the Fast Downward Vagrantfile for
+the desired [release](https://www.fast-downward.org/latest/releases). 
+
+If you want to use LP solvers see [LP Solvers](#lp_solvers).
+
+Otherwise, you can set up your virtual machine from the `<my-fast-downward-vm>` directory using:
+    
+    vagrant up
+
+#### LP solvers
+
+The !SoPlex LP solver is included automatically. If you want to also
+use the CPLEX LP solver within the planner, its installer file must
+be present in the directory `<path-to-lp-installers>`. As of Fast
+Downward 23.06, you will need CPLEX 22.1.1 (installer filename
+`cplex_studio2211.linux-x86-64.bin`). 
+
+To set up your virtual machine from the `<my-fast-downward-vm>` directory as follows:
 
 ``` bash
-# Set up the VM. Only run this one.
-mkdir my-fast-downward-vm
-cp Vagrantfile my-fast-downward-vm/
-cp domain.pddl problem.pddl my-fast-downward-vm/
-# Skip next line if you don't need LP support.
-export DOWNWARD_LP_INSTALLERS=/path/to/lp/installers
-cd my-fast-downward-vm
+export DOWNWARD_LP_INSTALLERS=<path-to-lp-installers>
+
 vagrant up
-# The VM is now set up.
-# You can now safely delete the LP installers and unset the environment variable.
+```
+
+You can now safely delete the LP installers and unset the environment variable.
+
+### Vagrant Usage
+You can run the planner from *within* the vagrant VM as follows:
+
+    downward/fast-downward.py <your-planner-options>
+
+Assume you want to solve the gripper planning task `prob01.pddl` from the [Fast Downward benchmarks](https://github.com/aibasel/downward-benchmarks) using A* search with the LM-cut heuristic. To do so you can run the following:
+
+``` bash
+# From the <my-fast-downward-vm> directory 
 
-# Log into the VM and run the planner.
+# Upload the benchmarks to the benchmarks directory on your vagrant machine
+vagrant upload <path-to-benchmarks> benchmarks/
+
+# Log into the VM.
 vagrant ssh
-downward/fast-downward.py --alias lama-first /vagrant/domain.pddl /vagrant/problem.pddl
+
+# Run the planner.
+downward/fast-downward.py /vagrant/benchmarks/gripper/prob01.pddl --search "astar(lmcut())"
 
 # Log out from the VM.
-exit
+logout
+
+# Shut down the machine
+vagrant halt
 ```
 
-## Source code
+Here `<path-to-benchmarks>` refers to your local benchmarks directory containing the [gripper benchmarks](https://github.com/aibasel/downward-benchmarks/tree/master/gripper).
+It can be specified as an absolute path, relative path (only as of Docker Engine version 23) or using a predefined path variable, e.g. `$BENCHMARKS`.
```

Comment:
```suggestion
It can be specified as an absolute path, relative path (only as of Docker Engine version 23) or using a predefined path variable, e.g. `$DOWNWARD_BENCHMARKS`.
```
or remove the part about variables